spec/spec_helper.rb

```diff
@@ -3,6 +3,7 @@
 ENV["RAILS_ENV"] ||= "test"
 ENV["HIDE_COMMAND_OUTPUT"] = "true"
 ENV["DISABLE_INTERRUPT_TRAP"] = "true"
+ENV["DISABLE_VALIDATIONS"] = "true"
```

Comment:
We don't want to run validations before each command in the tests because some validations are intended to fail and we want to check those separately.